doc/changelog.md

```diff
@@ -90,6 +90,12 @@
 
 <h3>Improvements</h3>
 
+* An optional argument `use_device_shots` is introduced in `PartialProbs` to enable computing analytical probabilities mid-circuit even when the device has finite shots.
+  [(#801)](https://github.com/PennyLaneAI/catalyst/pull/801)
+
+* Catalyst uses the `collapse` method of Lightning simulators in `Measure` to select a state vector branch and normalize.
+  [(#801)](https://github.com/PennyLaneAI/catalyst/pull/801)
+
```

Comment:
The first part is now obsolete. However, I would say this pr has two kinds of changes:
- a bugfix: use analytic shots when computing probs for mcms
- an internal change: use collapse from lightning

Reply:
Thanks, I haven't familiarized with the Catalyst changelog sections. I'll update the changelog.